test/unit/test_sap_cli_package.py

```diff
@@ -143,6 +147,46 @@ def test_with_recursion(self, fake_walk):
                                                  call('$VICTORY_TESTS/ZCL_TESTS'),
                                                  call('$VICTORY_DOC/')])
 
+class TestPackageStat(unittest.TestCase):
+
+    def test_stat_existing(self):
+
+        conn = Connection([Response(status_code=200, headers={}, text=GET_PACKAGE_ADT_XML)])
+
+        args = parse_args('stat', '$IAMTHEKING')
+        #sap.cli.package.stat(connection, args)
+
+        with patch('sap.cli.package.print', newcallable=StringIO) as fake_print:
+            #self.assertEqual(args.execute(conn, args), sap.cli.core.ExitCode.OK)
+            args.execute(conn, args)
+
+        self.assertEqual(len(conn.execs), 1)
+
+        if False:
+            self.assertEqual(fake_print.mock_calls, [
+                call('Name                   :$IAMTHEKING'),
+                call('Active                 :active'),
+                call('Application Component  :-'),
+                call('Software Component     :LOCAL'),
+                call('Transport Layer        :'),
+                call('Package Type           :development'),
+            ])
+
+    def test_stat_not_found(self):
+
+        conn = Connection([Response(status_code=404, headers={'content-type': 'application/xml'}, text=GET_PACKAGE_ADT_XML_NOT_FOUND)])
```

Comment:
Once you have the variable GET_PACKAGE_ADT_XML_NOT_FOUND holding the ADT form XML, then you don't need to patch the module sap.adt.ADTObject.fetch.